automation/src/test/resources/startTunnel.sh

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env bash
+set -e -x
```

Comment:
@jdcanas were you saying that this is what is causing the GCP automation test to fail?

Reply:
no, this line https://github.com/DataBiosphere/leonardo/pull/4581/files#diff-d176682250d61f7b1e0d9d9513e4a47c5ccfd02da8a27752c46e38d9b8825984R2

Reply:
same premise though. Essentially it is expected that azure sign in won't work if the tests are being run for google. You can either
1) add an arg so the script is aware of gcp/google (may be a bit annoying since I believe it is triggered by something in `terra-github-workflows`
2) `| true` the lines that are now failing (that always failed for google) so the script doesn't exit. This could possibly hide a real issue though... (i.e. the creds rotating)

Reply:
How fun, yeah it is called here: https://github.com/broadinstitute/terra-github-workflows/blob/28726262ceb146ca26f44672f354aa00015f2464/.github/actions/run-leo-swat-tests/action.yaml#L425

Reply:
By variables you mean these ones?
LEO_AZURE_CLIENT_ID
LEO_AZURE_CLIENT_SECRET
LEO_AZURE_TENANT_ID
LEO_AZURE_SUBSCRIPTION_ID